lib/OpenQA/WebAPI/Controller/API/V1/Table.pm

```diff
@@ -320,4 +282,51 @@ sub destroy {
     $self->render(json => {result => int($ret)});
 }
 
+=over 4
```

Comment:
Hm, for other POD in the same file and also others we use a different style for method headers. Would you mind using the same style here? E.g. head2, I think

Reply:
I just checked, sorry, disregard my comment. The file already uses the same style in another place. leave everything as is :)

Reply:
I don't think we actually have a consistent documentation style for openQA. Might make sense to agree on a standard though.